tests/integration/service/test_feature_table_cache.py

```diff
@@ -0,0 +1,293 @@
+"""
+Integration test for feature table cache
+"""
+import time
+
+import pandas as pd
+import pytest
+from sqlglot import parse_one
+
+from featurebyte import FeatureList
+from featurebyte.enum import InternalName
+from featurebyte.query_graph.sql.common import sql_to_string
+from tests.util.helper import create_observation_table_from_dataframe
+
+
+@pytest.fixture(name="event_view")
+def event_view_fixture(event_table):
```

Comment:
Are these fixtures copied from the event view integration tests? Would be good to move them to conftest to avoid duplicating code unnecessarily.